<?xml version="1.0"?>
<launch>

			<arg name="simulation"               default="false"/> 
			<arg name="just_visualize"           default="false"/> 
			<arg name="r_arm_ns"                 default="right_arm"/> 
			<arg name="l_arm_ns"                 default="left_arm"/>

			<arg name="r_robot_base_frame"       default="/calib_right_arm_base_link"/>
			<arg name="l_robot_base_frame"       default="/calib_left_arm_base_link"/>

			<arg name="model_path"               default="$(find bimanual_motion_planner)../bimanual-task-models/scooping_task"/>

		  <node pkg="tf" type="static_transform_publisher" name="task_frame" args="-0.403 -0.426 0.013 0 0 0 1  /world_frame /task_frame  100"/>


		  <node name="bimanual_plan2ctrl" pkg="bimanual_motion_planner" type="bimanual_action_server_node" output="screen" respawn="false">	  	
		  	<param name="simulation" 			 		value="$(arg simulation)" />
		  	<param name="just_visualize" 			 	value="$(arg just_visualize)" />
		  	<param name="right_robot_frame" 		    value="$(arg r_robot_base_frame)" />
		  	<param name="left_robot_frame" 		     	value="$(arg l_robot_base_frame)" />
		  	<param name="model_dt" 				 		value="0.005" />
		  	<param name="reachingThreshold" 	 		value="0.005" />
		  	<param name="orientationThreshold"   		value="0.05" />
			<param name="model_base_path"        	    value="$(arg model_path)" />
			<param name="r_topic_ns"        		    value="$(arg r_arm_ns)" />
			<param name="l_topic_ns"        		    value="$(arg l_arm_ns)" />
			<param name="wait_for_force_right_arm" 		value="true"/>
			<param name="wait_for_force_left_arm" 		value="true"/>
			<param name="task_id"						value="2"/>
		  </node>


</launch>
